bin/resty

```diff
@@ -475,6 +481,8 @@ Options:
     --main-include path Include the specified file in the nginx main configuration block
                         (multiple instances are supported).
 
+    --resolver ip       Specify the name resolver(s) (multiple instances are supported)
```

Comment:
Shall we use all upper case for the placeholder value `ip`? That is, `--resolver IP`.

I know we also need to fix `--main-include path` above. But let's do it in a separate PR :)